cmd/root.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "container-runtime",
+	Short: "container-runtime is a simple container runtime implementation",
+}
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		os.Exit(1)
```

Comment:
Maybe print info why exit failure

Reply:
If I implement the thing from above, I think it's sufficient to print the error in `main`

Reply:
yes, then main should be the main (only) place for printing errors :slightly_smiling_face:

Reply:
hmm can't mark conversation as resolved